trusted_application_demo/client_application/aes_certify_messaging/publisher/CMakeLists.txt

```diff
@@ -0,0 +1,56 @@
+cmake_minimum_required(VERSION 3.5)
+project(aes_publisher_certify)
+
+# Default to C++14
+if(NOT CMAKE_CXX_STANDARD)
+  set(CMAKE_CXX_STANDARD 14)
+endif()
+
+set(BINARY_NAME aes_publisher_certify)
+add_compile_options(-Wall -Wextra -Wpedantic)
+
+# Trusted application libraries
+if((NOT DEFINED OPTEE_CLIENT_EXPORT) OR (NOT IS_DIRECTORY ${OPTEE_CLIENT_EXPORT}))
+  message(WARNING "OPTEE_CLIENT_EXPORT must be defined to compile ${PROJECT_NAME} -- Skipping this package")
+  return()
+endif()
+
+link_directories(${OPTEE_CLIENT_EXPORT}/lib)
+include_directories(${OPTEE_CLIENT_EXPORT}/include)
+
+find_package(ament_cmake REQUIRED)
+find_package(rclcpp REQUIRED)
+find_package(fastrtps REQUIRED)
+find_package(fastcdr REQUIRED)
```

Comment:
I don't see any `fastcdr` symbol used in this package. Is this dependency necessary?

Reply:
Same for Fast-RTPS, is there any code specific to Fast-RTPS in this PR ?

Reply:
Those packages should not be here, I forgot to remove them :)